docs/graphql/core/guides/postgres/import-data.rst

```diff
@@ -0,0 +1,67 @@
+.. meta::
+   :description: import data from csv in postgres
+   :keywords: hasura, docs, postgres, import, data
+
+.. _postgres_import_data_from_csv:
+
+Import data from csv into database
+===================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+-------------
+
+CSV files are plain text files that makes it easy for any developer to create and add data. 
+Also, the data are easy to import into any database. 
+
+Step 1: Create a Postgres instance
+------------------------------------
+
+Create a postgres database and run the command below on the console/terminal to create a table callled ``profile``
+with the following set of fields.
+
+.. code-block:: bash
+
+  CREATE TABLE profile (firstName text, lastName text, email varchar);
+
+Step 2: Let's create a new CSV file that holds some data
```

Comment:
We assume that people already have a CSV file that they want to import. If they start from scratch, it makes no sense to create a CSV. They could add the data directly to Hasura / Postgres. 

So we can say something like: Assume you have a CSV file with data that you want to import into your Postgres DB, like the following. (not the exact wording but I think you get what I mean)